### DISCUSSION TITLE
Mobile samples

### DISCUSSION THREAD

**Discussion by DuarteMPG** [rds1983/Myra#389] [Q&A]
Hello friends!

I'm looking into using Myra for a mobile project. Are there any samples targeting Android/iOS out there I could get inspiration from?
Currently Android works fine, but iOS doesn't seems to work because of the AOT restrictions.

Thanks!

**Comment by MeGaL0DoN** [comment]
I don't know about IOS AOT, but on windows AOT works if you add this to rd.xml:  `<Assembly Name="Myra" Dynamic="Required All" />`
